### PR TITLE
Bookworm unleashed; Bullseye no longer "stable"

### PR DESCRIPTION
**Pull request by ericboehs**
- Dockerfile: Use bookworm instead of stable/unstable/testing
- Dockerfile: DRY up ruby image version

Resolves docker build failures for master (Jenkins/RIs).

### CODE REVIEW

**Review by ryan-mcneil — APPROVED** [review]
🤞

**Comment by ph-One** [comment]
I'm not entirely sure why we want to repeat ourselves [instead of using a variable for the release](https://github.com/department-of-veterans-affairs/vets-api/pull/12962/files), but I'm not going to fight the DRY battle here.

**Review by ryan-mcneil — DISMISSED** [review]
🚀

File: Dockerfile [hunk 1]
```diff
@@ -14,23 +11,17 @@ RUN find modules -type f ! \( -name Gemfile -o -name "*.gemspec" -o -path "*/lib
...
 
 ARG userid=993
 SHELL ["/bin/bash", "-c"]
 RUN groupadd -g $userid -r vets-api && \
     useradd -u $userid -r -m -d /srv/vets-api -g vets-api vets-api
-RUN echo 'APT::Default-Release "stable";' >> /etc/apt/apt.conf.d/99defaultrelease
-RUN mv /etc/apt/sources.list /etc/apt/sources.list.d/stable.list
-RUN echo "deb http://ftp.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list.d/testing.list
-RUN echo "deb http://deb.debian.org/debian unstable main" >> /etc/apt/sources.list.d/unstable.list
+RUN echo "deb http://ftp.debian.org/debian bookworm main contrib non-free" >> /etc/apt/sources.list.d/bookworm.list
```
Comment (ericboehs) [inline Dockerfile:20]: Once Ruby releases a bookworm version, we can remove this line and remove `-t bookworm` from line 22. I'd expect a release in the next week.

**Review by rmtolmach — APPROVED** [review]

**Pull request closed without merging**